internal/kibana/policies.go

```diff
@@ -72,6 +72,62 @@ func (c *Client) GetPolicy(policyID string) (*Policy, error) {
 	return &resp.Item, nil
 }
 
+// GetRawPolicy fetches the given Policy with all the fields in the Ingest Manager.
+func (c *Client) GetRawPolicy(policyID string) (json.RawMessage, error) {
+	statusCode, respBody, err := c.get(fmt.Sprintf("%s/agent_policies/%s", FleetAPI, policyID))
+	if err != nil {
+		return nil, errors.Wrap(err, "could not get policy")
+	}
+
+	if statusCode != http.StatusOK {
+		return nil, fmt.Errorf("could not get policy; API status code = %d; response body = %s", statusCode, respBody)
+	}
+
+	var resp struct {
+		Item json.RawMessage `json:"item"`
+	}
+
+	if err := json.Unmarshal(respBody, &resp); err != nil {
+		return nil, errors.Wrap(err, "could not convert policy (response) to JSON")
+	}
+
+	return resp.Item, nil
+}
+
+// ListRawPolicy fetches all the Policies in the Ingest Manager.
+func (c *Client) ListRawPolicy() ([]json.RawMessage, error) {
+	itemsRetrieved := 0
+	currentPage := 1
+	var items []json.RawMessage
+	var resp struct {
+		Items   []json.RawMessage `json:"items"`
+		Total   int               `json:"total"`
+		Page    int               `json:"page"`
+		PerPage int               `json:"perPage"`
+	}
+
+	for finished := false; !finished; finished = itemsRetrieved == resp.Total {
```

Comment:
Nit.Would this be equivalent?
```suggestion
	for itemsRetrieved < resp.Total {
```

Reply:
At least as it is coded, it is not equivalent. 
The first iteration of the loop resp.Total has the default value (zero) since there was no query run against fleet yet. This struct gets the value in each iteration. With that suggestion, the loop never gets executed.

Using that for loop in that way, I tried to mimic a do-while loop.